src/button/template/componentTemplate.jsx

```diff
@@ -228,27 +228,102 @@ function renderButton({ size, label, color, locale, branding, multiple, layout,
     );
 }
 
-function renderTagline({ label, tagline, color, locale, multiple, env, cards } : { label : string, color : string, tagline : boolean, locale : LocaleType, multiple : boolean, env : string, cards : $ReadOnlyArray<string> }) : ?JsxHTMLNode {
+function renderTagline({ label, tagline, color, locale, multiple, env, cards, checkoutCustomization } : { label : string, color : string, tagline : boolean, locale : LocaleType, multiple : boolean, env : string, cards : $ReadOnlyArray<string>, checkoutCustomization : ?CheckoutCustomizationType }) : ?JsxHTMLNode {
+    const delay = 0.2;
 
+    if (__WEB__) {
+        return (
+            <div>
+                <style innerHTML={ `
```

Comment:
Can we keep this in it's own component/function? Given we know we'll need to use it elsewhere